src/DataTable/useTable.ts

```diff
@@ -207,6 +172,66 @@ export function useTable<Data extends UniqueRow>({
   }
 }
 
+function getInitialSortState<Data extends UniqueRow>(
+  columns: Array<Column<Data>>,
+  initialSortColumn?: string | number,
+  initialSortDirection?: Exclude<SortDirection, 'NONE'>,
+) {
+  if (initialSortColumn !== undefined) {
+    const column = columns.find(column => {
+      return column.id === initialSortColumn || column.field === initialSortColumn
+    })
+
+    if (column === undefined) {
+      if (__DEV__) {
```

Comment:
Is this a good place to use the `warning` utility you created?

Reply:
@colebemis exactly 👍